dependencies.gradle

```diff
@@ -48,8 +48,10 @@ ext {
   commonsCodecVersion = '1.15'
   commonsCollections4Version = '4.4'
   guavaVersion = '31.1-jre'
+  byteBuddyVersion = '1.14.4'
 
   // Logging
+  logbackVersion = '1.2.12'
```

Comment:
according to their homepage we should use 1.4.x for jdk11+
https://logback.qos.ch/download.html

Reply:
You are right about the JDK11 stuff, but this is also mentioned there:
> supporting Jakarta EE (jakarta.* namespace)
We still depend on Java EE. Jakarta EE will be needed for Spring Boot 3.x / Spring 6 anyway, so we will then be able migrate LogBack as well. (Along with some other libraries like Swagger 3 / OpenAPI, Hibernate, Quartz etc.)